homeassistant/components/samsungtv/media_player.py

```diff
@@ -103,78 +108,32 @@ def __init__(self, config_entry, on_script):
             "port": config_entry.data.get(CONF_PORT),
             "host": config_entry.data[CONF_HOST],
             "timeout": 1,
+            "token_file": config_entry.data[CONF_TOKEN_FILE],
         }
+        self._bridge = SamsungTVBridge.get_bridge(self._config)
+
+    def acces_denied(self):
+        """Access denied callbck."""
+        LOGGER.debug("Access denied in getting remote object")
```

Comment:
Should we use `info` at least?

Reply:
No idea honestly. Now it's just a way to trigger the reauth.